llvm/lib/SYCLLowerIR/LowerESIMDVLoadVStore.cpp

```diff
@@ -0,0 +1,142 @@
+//===- LowerESIMDVLoadVStore.cpp - lower vload/vstore to load/store -------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// convert vload/vstore to load/store if they are not for genx_volatile.
+//
+// File scope simd variables marked with genx_volatile attribute want
+// guarranteed allocation in register file, therefore we use vload/vstore
+// instead of load/store, so they won't be optimized away by llvm.
+//
+// For ordinary simd variables, we do not need to protect load/store. But
+// there is no good way to do this in clang. So we need this pass in the
+// end of module passes to separate the cases that we need vload/vstore vs.
+// the cases that we do not need vload/vstore
+//
+//===----------------------------------------------------------------------===//
+
+#define DEBUG_TYPE "loweresimdvloadvstore"
+
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/GenXIntrinsics/GenXIntrinsics.h"
+#include "llvm/IR/Function.h"
+#include "llvm/IR/IRBuilder.h"
+#include "llvm/IR/InstIterator.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/IR/IntrinsicInst.h"
+#include "llvm/IR/Module.h"
+#include "llvm/InitializePasses.h"
+#include "llvm/Support/Debug.h"
+#include "llvm/Support/raw_ostream.h"
```

Comment:
Probably some includes are not needed.

Reply:
Which includes are not needed. Please be specific

Reply:
Review process doesn't work this way.
I pointed to "#include "llvm/Support/raw_ostream.h" which probably was used during debugging the code but is not needed in the published code. And you should check if other headers are needed.

Reply:
I will delete raw_ostream.h